outputs.tf

```diff
@@ -43,11 +43,16 @@ output "alarm" {
   value       = "${join("", aws_cloudwatch_metric_alarm.default.*.id)}"
 }
 
-output "additional_eni_ids" {
-  description = "Map of ENI with EIP"
+output "additional_eni_public" {
+  description = "Map IDs of ENI with public IPs"
   value       = "${zipmap(aws_network_interface.additional.*.id, aws_eip.additional.*.public_ip)}"
 }
 
+output "additional_eni_private" {
```

Comment:
Why do we need this for?

Reply:
for example it need in sth for template rendering